docs/guides/integration-prebidjs.md

```diff
@@ -0,0 +1,193 @@
+
+The publisher is responsible for normalizing the phone number. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md#phone-number-normalization) for details on phone number normalization.
+
+### Configure for hashed email
+
+Configure the UID2 module with a hashed email address:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        emailHash: 'eVvLS/Vg+YZ6+z3i0NOpSXYyQAfEXqCZ7BTpAjFUBUc=',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will encrypt the hash before sending it to the UID2 service.
+
+The publisher is responsible for normalizing and hashing the email address. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for details on email address normalization and hashing.
+
+### Configure for hashed phone
+
+Configure the UID2 module with a hashed phone number:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        phoneHash: 'eVvLS/Vg+YZ6+z3i0NOpSXYyQAfEXqCZ7BTpAjFUBUc=',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will encrypt the hash before sending it to the UID2 service.
+
+The publisher is responsible for normalizing and hashing the phone number. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for details on phone number normalization and hashing.
+
+## Check the integration
+
+Check that Prebid.js and the UID2 module are configured correctly by calling `pbjs.getUserIds().uid2`. If everything is working, this will return the UID2 token.
+
+## Reconfigure the UID2 module as required
+
+If a user is visiting your site frequently, the UID2 module will be able to continually refresh the UID2 token.  However there are cases where this is not possible.
+
+A UID2 token can only be refreshed for a certain amount of time. If a user returns to your site after this amount of time, you will need to re-configure the UID2 module.
+
+To check if the UID2 module needs to be reconfigured:
+
+```js
+TODO
+```
+
+## Optional: Set the UID2 API Base URL
+
+The UID2 module uses a default UID2 API base URL of `https://prod.uidapi.com`. To use a different base URL, set the `params.uid2ApiBase` parameter when configuring the UID2 module:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        uid2ApiBase: baseUrl,
+        // ...
+      }
+    }]
+  }
+});
+```
+
+Refer to [Environments](../getting-started/gs-environments.md) for the list of possible base URLs.
+
+The default URL points to a server in the USA. Publishers should take into account where their readers are based, and consider choosing a different base URL in order to reduce latency.
```

Comment:
A second thought, we should bold or mark this sentence as important note and probably move it above the code example so people can notice this more easily

Reply:
Maybe also rephrase along the line of:

_By default, UID2 module will make API calls to a server in the USA. Publishers should take into account where their readers are based, and consider choosing a different base URL in order to reduce latency._